content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -771,7 +771,7 @@ validations are not supported by ratcheting under the implementation in Kubernet
 
 ## Validation rules
 
-{{< feature-state state="beta" for_k8s_version="v1.25" >}}
+{{< feature-state state="stable" for_k8s_version="v1.29" >}}
```

Comment:
You must also update it on the [Feature Gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) page.

Reply:
Also, acc to [k8s/k8s #2876](https://github.com/kubernetes/enhancements/issues/2876) and [k8s/k8s #121164](https://github.com/kubernetes/kubernetes/issues/121164) it is not implemented as GA, once it will GA then we can update the corresponding docs.

Reply:
Hold it, until it is implemented in the code.
/hold

Please remove the hold label once it is done.